package/ios/Core/CameraSession+Photo.swift

```diff
@@ -63,7 +63,8 @@ extension CameraSession {
       let enableShutterSound = options["enableShutterSound"] as? Bool ?? true
 
       // depth data
-      photoSettings.isDepthDataDeliveryEnabled = photoOutput.isDepthDataDeliveryEnabled
+      photoSettings.isDepthDataDeliveryEnabled = photoOutput.isDepthDataDeliveryEnabled && photoOutput.isDepthDataDeliverySupported
```

Comment:
Is this change really required? Isn't `isDepthDataDeliveryEnabled` only `true` when `isDepthDataDeliverySupported` is also `true`? I can't see a scenario where it is enabled but not supported 😅 

Maybe the method throws an error if you try to set any value to `photoSettings.isDepthDataDeliveryEnabled`, even if it's `false`? In that case we **MIGHT** need to wrap it under an if instead:

```swift
if photoOutput.isDepthDataDeliverySupported {
  photoSettings.isDepthDataDeliveryEnabled = photoOutput.isDepthDataDeliveryEnabled
}
```

Reply:
Apparently yes, you are right. It wasn't mentioned in the isDepthDataDeliveryEnabled documentation page, but it was mentioned at the bottom of the isDepthDataDeliverySupported. I guess you can scrap the PR then